doc/services/retention/index.rst

```diff
@@ -182,7 +182,7 @@ which will allow rebooting directly into the serial recovery mode by using:
 	#include <zephyr/sys/reboot.h>
 
 	bootmode_set(BOOT_MODE_TYPE_BOOTLOADER);
-	sys_reboot(0);
+	sys_reboot(SYS_REBOOT_DEFAULT);
```

Comment:
It will preserve RAM contents since the default is the functional reset, described in https://github.com/zephyrproject-rtos/zephyr/pull/60920#discussion_r1281476723